From: Michael Froman <mfroman@mozilla.com>
Date: Tue, 10 Jun 2025 17:41:00 -0500
Subject: Bug 1971201 - (fix-8fc919730c) revert .gn change for allowlist until
 we can update our gn executable in .mozbuild

Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/f71a90014b54ba17e14333579b23644eba89d075
---
 .gn | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gn b/.gn
index fa6a5e85d8..4c9b95c31b 100644
--- a/.gn
+++ b/.gn
@@ -32,10 +32,10 @@ no_check_targets = [
   "//third_party/googletest:gtest",
 ]
 
-# These are the list of GN files that run exec_script. This allowlist exists
+# These are the list of GN files that run exec_script. This whitelist exists
 # to force additional review for new uses of exec_script, which is strongly
 # discouraged except for gypi_to_gn calls.
-exec_script_allowlist = build_dotfile_settings.exec_script_allowlist +
+exec_script_whitelist = build_dotfile_settings.exec_script_whitelist +
                         [ "//libwebrtc/build_overrides/build.gni" ]
 
 # Normally, we'd use 'if (!build_with_mozilla)', but that flag isn't
